pkg/deploy/api/validation/validation.go

```diff
@@ -37,7 +37,7 @@ func ValidateDeploymentConfig(config *deployapi.DeploymentConfig) fielderrors.Va
 	if config.Spec.Replicas < 0 {
 		allErrs = append(allErrs, fielderrors.NewFieldInvalid("spec.replicas", config.Spec.Replicas, "replicas cannot be negative"))
 	}
-	if config.Spec.Selector == nil || len(config.Spec.Selector) == 0 {
+	if len(config.Spec.Selector) == 0 {
```

Comment:
> The dc selector will be propagated in the pod labels. So a pod owned by a dc will have both the dc selector and any podtemplate labels as its label set.

Huh, didn't see that coming.  Is that well known via something other than swagger?